src/components/Navbar/Navbar.jsx

```diff
@@ -2,23 +2,25 @@ import React from "react";
 
 import s from "./Navbar.module.css";
 
+import { NavLink } from "react-router-dom";
+
 const Navbar = () => {
   return (
     <nav className={s.nav}>
       <div className={`${s.item} ${s.active}`}>
-        <a href="#">Profile</a>
+        <NavLink to="/profile" className={s.activeLink}>Profile</NavLink>
       </div>
       <div className={`${s.item} ${s.active}`}>
-        <a href="#">Messages</a>
+        <NavLink to="/dialogs">Messages</NavLink>
       </div>
       <div className={`${s.item} ${s.active}`}>
-        <a href="#">News</a>
+        <NavLink to="/news">News</NavLink>
```

Comment:
Consider creating a folder "core" at the root level (e.g. src/core) and a "constants.js" file in that folder.
In this file you will keep the common reusable constants of your project.
As for the current line, consider creating object "I18Y" inside the "constants.js" file, than inside this object - another object named "EN" (short for English) and put "NEWS" as a key and the "News" string as value insider EN object.
`const I18Y = {`
`   EN: {`
`      NEWS: "News,`
`    },`
`};`
Do this for EVERY such string in the whole project - find them and put them inside I18Y object under the respective name.

Reply:
Done